FHEM/00_SIGNALduino.pm

```diff
@@ -1348,6 +1348,14 @@ sub SIGNALduino_CheckVersionResp {
 			$hash->{logMethod}->($name, 5, "$name: CheckVersionResp, cc1101 available");
 			SIGNALduino_Get($hash, $name,"ccconf");
 			SIGNALduino_Get($hash, $name,"ccpatable");
+		} else {
+			# connect device without cc1101 to port where a device with cc1101 was previously connected (example DEF with /dev/ttyUSB0@57600) #
+			$hash->{logMethod}->($hash, 5, "$name: CheckVersionResp, delete old READINGS from cc1101 device");
+			delete($hash->{cc1101_available}) if exists( $hash->{cc1101_available} );
```

Comment:
Laut best practice soll man das If nur nach hinten stellen, wenn der Programmablauf verändert wird.

Daher hatte ich den Vorschlag gemacht, die Aktion in einen If Block zu setzen :)
Das If könnte man aber auch weglassen oder gibt es einen Fehler wenn der Key nicht existiert?

Reply:
Ich habe nun einen if Block draus gemacht um das If nicht nach hinten zu verfrachten.

Da ich es nicht mit dem HashWert löschen gerade testen kann, habe ich das IF belassen.